src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaTestData.cs

```diff
@@ -32,6 +29,396 @@ public static class SlhDsaTestData
+                """,
+                """
+                MIGTAgEAMAsGCWCGSAFlAwQDGASBgKj+fFB/g4L/WGmuAJYSC+ACUn8gPnZl8mjc
+                KVrlE+MOdNJ0q8c5T9bc9fwPwcJFTbU+yOTvzA8a+0iTLcNqUS90B0nPuRiei8x8
+                jACsbj9hCPHkCLvx6kPMuadrz80gw3SEjT3Ls4Zxb93UBPur4jAvzOlXrgU/Y23f
+                OF228SUN
+                """,
+                """
+                MFAwCwYJYIZIAWUDBAMYA0EAdAdJz7kYnovMfIwArG4/YQjx5Ai78epDzLmna8/N
+                IMN0hI09y7OGcW/d1AT7q+IwL8zpV64FP2Nt3zhdtvElDQ==
+                """,
+                """
+                MIH2MFEGCSqGSIb3DQEFDTBEMCMGCSqGSIb3DQEFDDAWBBBbUw8TIria5X9svVdP
+                eIfJAgInEDAdBglghkgBZQMEARYEEC5WKDwQteQdhCOBs6p2WtQEgaDmxL8MnYfe
+                Egu+kVs2AzJS7r2k/aqF5eOeTqblZ635iqhzg53Mqa4kr+/oUu2jE3Kz8thevGNa
+                urgBqetWejb4Veu4g1/MHxrPp1x6cAZ8FrxS1Vt2969ONYhUeMYlWoRa1BqdM1Es
+                vnOjBI6rNUkH9pPTtkeU08T3x6mw/El5r5oGvcLmXSMRaKtjMP85fkesZBZ4uCED
+                RJMHPtoI+GZJ
+                """,
+                "PLACEHOLDER",
+                new PbeParameters(
+                    encryptionAlgorithm: PbeEncryptionAlgorithm.Aes192Cbc,
+                    hashAlgorithm: HashAlgorithmName.SHA1,
+                    iterationCount: 10000
+                )),
+            new(Id: 10,
+                SlhDsaAlgorithm.SlhDsaSha2_256f,
+                """
+                F59CF04EC5B2331F7ABB8DFB01CD5B53E543D2A7CC95A91713D4CBC970381A26FA87B5D266D42F4AE7F761D761606A227B0A52AA4860E58969946DA0B754710064726B6F65B9DFCB6222275546F3462618D360368823D25619E5E7D52CE4FCC14C7D530B15F1E1373288A493BFF1C7D4814DDC74F7CF0F21A509DAD4CD5517C9
+                """,
+                """
+                MIGTAgEAMAsGCWCGSAFlAwQDGQSBgPWc8E7FsjMferuN+wHNW1PlQ9KnzJWpFxPU
+                y8lwOBom+oe10mbUL0rn92HXYWBqInsKUqpIYOWJaZRtoLdUcQBkcmtvZbnfy2Ii
+                J1VG80YmGNNgNogj0lYZ5efVLOT8wUx9UwsV8eE3Moikk7/xx9SBTdx0988PIaUJ
+                2tTNVRfJ
+                """,
+                """
+                MFAwCwYJYIZIAWUDBAMZA0EAZHJrb2W538tiIidVRvNGJhjTYDaII9JWGeXn1Szk
+                /MFMfVMLFfHhNzKIpJO/8cfUgU3cdPfPDyGlCdrUzVUXyQ==
+                """,
+                """
+                MIIBBTBgBgkqhkiG9w0BBQ0wUzAyBgkqhkiG9w0BBQwwJQQQqeq71JYbqs/xO7es
+                /377AAIDD0JAMAwGCCqGSIb3DQIJBQAwHQYJYIZIAWUDBAEqBBA/Cb/Rm7HqEnfO
+                qvtxeJBOBIGgu1pjBQPoT1SylVZLQ6p/h8v6ehle+LZDkNb/G7PDosVKV+7u//oL
+                E8FzEUfJtRXPc5vGFnaF9B9jGm9ritrECHfKP84piUJq6PPpOoitZzjC1l1vGIoq
+                Cy+JgPX1xkQokBc2r7yD0UVMmUaTv1/ePXw9xfnfOyslJ53a/rbahDFSPcOAyS5V
+                ddETtlKUJKnkIkjlf4n3HWvY9+QY9DrTlg==
+                """,
+                "PLACEHOLDER",
+                new PbeParameters(
+                    encryptionAlgorithm: PbeEncryptionAlgorithm.Aes256Cbc,
+                    hashAlgorithm: HashAlgorithmName.SHA256,
+                    iterationCount: 1000000
```

Comment:
Likewise, another million iteration count. If we actually do a million iterations, it will show up in CI. We should try to keep the iteration counts below 1000, probably.